src/gpu/intel/jit/gemm/jit_gemm_pd.cpp

```
@@ -169,10 +169,24 @@ bool jit_gemm_pd_t::quant_attr_2d(int arg, const quant_entries_t &attr) const {
     return false;
 }
 
-int jit_gemm_pd_t::quant_attr_cmask(
-        int arg, const quant_entries_t &attr) const {
-    assert(utils::one_of(arg, DNNL_ARG_A, DNNL_ARG_B, DNNL_ARG_C));
-    if (!attr.has_default_values(arg)) { return attr.get_mask(arg); }
+int jit_gemm_pd_t::quant_attr_ndims(
+        const quant_entry_t &attr, const memory_desc_t &d) const {
+    if (attr.has_default_values()) return -1;
+    int mask = quant_attr_cmask(attr);
+    if (mask == 0) return 0;
+    if (attr.has_default_groups()) return mask > 0;
+    int count = 0;
+    for (int i = 0; i < d.ndims; ++i) {
+        if ((mask & (1 << i))
+                && ((i < batch_dims() && d.dims[i] > 1)
+                        || (d.dims[i] / attr.get_group(i - batch_dims()) > 1)))
+            ++count;
+    }
+    return count;
+}
+
+int jit_gemm_pd_t::quant_attr_cmask(const quant_entry_t &attr) const {
+    if (!attr.has_default_values()) { return attr.get_mask(); }
     return -1;
```

Generally, `-1` is a bad mask as it doesn't allow to work with bits as opposed to `INT_MIN` ised as a default values in attributes, I'd suggest to eliminate this method in a favor of library quant_entry API.

---

Updated default mask to `INT_MIN` in primitive and dropped this method, thanks.